auth/server/src/main/java/com/seagates3/parameter/validator/AccountParameterValidator.java

```diff
@@ -40,12 +40,27 @@
      */
     @Override
     public Boolean isValidCreateParams(Map<String, String> requestBody) {
-        if (!S3ParameterValidatorUtil.isValidEmail(requestBody.get("Email"))) {
-            return false;
+        Boolean isValidParameters = S3ParameterValidatorUtil.isValidEmail(
+                                                    requestBody.get("Email"));
+
+        if (isValidParameters) {
+            isValidParameters = S3ParameterValidatorUtil.isValidName(
+                                                requestBody.get("AccountName"));
         }
 
-        return S3ParameterValidatorUtil.isValidName(
-                requestBody.get("AccountName"));
+        if (isValidParameters) {
+            String accessKey = requestBody.get("AccessKey");
+            String secretKey = requestBody.get("SecretKey");
+
+            if (accessKey != null || secretKey != null) {
```

Comment:
This condition has to be &&

Reply:
If one of the key is not null and another is null, we still want to validate and send error to user that he has only provided one parameter and he should provide both. If we use && then it will not validate and isValidParameters will remain true.